Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -66,6 +67,224 @@ constexpr int MACH_O_POWERPC_750 = 9;
 
 const s64 GDB_UPDATE_CYCLES = 100000;
 
+const char* QUERY_XFER_TARGET = "qXfer:features:read:target.xml:";
+const char* QUERY_XFER_MEMORY_MAP = "qXfer:memory-map:read::";
+
+const char* TARGET_MEMORY_MAP_XML_NO_MMU =
+    "<memory-map version=\"1.0\">"
+    "<memory type=\"ram\" start=\"0x7E000000\" length=\"0x2000000\"/>"
+    "<memory type=\"ram\" start=\"0x80000000\" length=\"0x1800000\"/>"
+    "<memory type=\"ram\" start=\"0x90000000\" length=\"0x4000000\"/>"
+    "</memory-map>";
+
+const char* TARGET_MEMORY_MAP_XML_WITH_MMU =
+    "<memory-map version=\"1.0\">"
+    "<memory type=\"ram\" start=\"0x0\" length=\"0x10000000\"/>"
+    "</memory-map>";
```

Comment:
Is this correct?  Based on [wiibrew](https://wiibrew.org/wiki/Memory_map) MEM1 physically covers `0x00000000`-`0x017FFFFF`, and MEM2 physically covers `0x10000000`-`0x13FFFFFF`.  This mapping seems like it won't cover MEM2 at all.